### DISCUSSION TITLE
2.3.20 Alerts Tab Timeout

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#2406] [General] [answered]
Hello,

I just upgraded my grid to 2.3.20.

All devices are showing online and running version 2.3.20 under the GRID tab.

When I go to the ALERTS tab, all I get is a timeout.  Kibana shows me current data for all sections after upgrade.   Appears to just be the ALERTS tab that isn't working.

Am I missing something?

![image](https://user-images.githubusercontent.com/54154986/102817593-ac9b9e80-4395-11eb-93b0-6132316af9ed.png)

**Comment by defensivedepth** [comment]
Depending on how far back you were, you will have to run `soup` multiple times  - can you confirm that when you run `sudo soup` on the Manager, it says you are up to date?

Also, any errors from running `sudo salt-call state.highstate` on the Manager?

**Reply by stondino00** [reply]
My install was only one version back at 2.3.10.

Yes when I run soup again it says it's on the current version.

salt-call state.highstate shows the following at the end.


[root@SO-Manager log]# sudo salt-call state.highstate
[WARNING ] State for file: /etc/pki/ca.crt - Neither 'source' nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined, yet 'replace' was set to 'True'. As there is no source to replace the file with, 'replace' has been set to 'False' to avoid reading the file unnecessarily.
[WARNING ] State for file: /etc/pki/ca.crt - Neither 'source' nor 'contents' nor 'contents_pillar' nor 'contents_grains' was defined, yet 'replace' was set to 'True'. As there is no source to replace the file with, 'replace' has been set to 'False' to avoid reading the file unnecessarily.




Summary for local
--------------
Succeeded: 573 (changed=16)
Failed:      0
--------------
Total states run:     573
Total run time:   246.371 s

**Comment by jertel** [comment]
Do you recall if the alerts took more than 15 seconds to show up prior to the upgrade?

**Reply by stondino00** [reply]
They did take a few seconds but do not recall them taking that long.  I never did receive this error and I was seeing alerts prior to .20 update.

Should I extend the timeout past 15 seconds to see if that works?  How do I do that exactly?

**Reply by TOoSmOotH** [reply]
@stondino00 We found an issue and are in the process of creating a 2.3.21 to address it. It has to be with the websocket timeout value which is inside the SOC docker container.

**Reply by stondino00** [reply]
Ok cool.  That sounds good.  Thanks

**Reply by TOoSmOotH** [reply]
2.3.21 is out

**Comment by stondino00** [comment, marked as answer]
2.3.21 appears to have fixed the issue.  Thanks guys!
